README.md

```
@@ -10,6 +10,7 @@ Repository for code related to small tasks for supporting the RD service.
 | [DI-1189] | Create new PanelApp JSON file
 | [DI-435] | Find and merge VCFs for creation of a GRCh38 POP AF VCF
 | [DI-1294] | Remove any content from the PID fields in workbooks in a given path
+| [DI-1299] | Compare values between GRCh37 and GRCh38 for Somalier
```

Markdownlint (0.37.0)

Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe
Table pipe style

(MD055, table-pipe-style)